src/test/java/eu/elision/marketplace/services/DynamicAttributeServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package eu.elision.marketplace.services;
+
+import eu.elision.marketplace.domain.product.category.attributes.Type;
+import eu.elision.marketplace.web.dtos.DynamicAttributeDto;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+class DynamicAttributeServiceTest {
+
+    @Autowired
+    private DynamicAttributeService dynamicAttributeService;
+
+    @Test
+    void toDynamicAttributeTest() {
```

Comment:
You might be able to deduplicate this test function with `@ParameterizedTest` and/or `@CsvSource` attributes